plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx

```diff
@@ -57,7 +57,7 @@ export const ScaffolderPageContents = () => {
         pageTitleOverride="Create a New Component"
         title={
           <>
-            Create a New Component <Lifecycle alpha shorthand />
+            Create a New Component <Lifecycle shorthand />
```

Comment:
No lifecycle at all? Does this default to beta if you leave it out? Das cray

Reply:
Presumably you're not using `<Lifecycle />` at all if it's in production 😛

Reply:
Sure but, a great api it does not make :)

Reply:
No I noticed that but didn't want to change all that in one PR